doc/tired_support.md

```diff
@@ -0,0 +1,197 @@
+# Tired Supported
```

Comment:
i guess you mean tiered.

is this list meant only for security-related classification?

Reply:
oops